packages/blockchain-wallet-v4-frontend/src/scenes/plugin/common/FullScreenModal/ModalPortal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { createPortal } from 'react-dom'
+
+const createWrapperAndAppendToBody = (wrapperId: string) => {
+  const wrapperElement = document.createElement('div')
+  wrapperElement.setAttribute('id', wrapperId)
+  document.body.appendChild(wrapperElement)
+  return wrapperElement
+}
+
+export interface IModalPortalProps {
```

Comment:
shouldn't be exported

Reply:
It's a common practice to exporting interfaces by default in case we want to create a new component that extends this one. Maybe in this project it's not necessary, so ok, I can remove it (here and in other places)